eks/cdk/fis.py

```diff
@@ -110,6 +107,42 @@ def __init__(self, app: core.App, id: str, props, **kwargs) -> None:
             tags = {'Name': 'Disk stress on EKS nodes'}
         )
 
+        # chaos-mesh pod-kill
+        aws_fis.CfnExperimentTemplate(
+            self, "eks-chaos-mesh-disrupt-pod",
+            description = "Disrupt pod",
+            role_arn = props['fis_role'].role_arn,
+            targets = {'eks-cluster': aws_fis.CfnExperimentTemplate.ExperimentTemplateTargetProperty(
+                resource_type = "aws:eks:cluster",
+                resource_arns = [f"{props['eks'].cluster_arn}"],
+                selection_mode = "ALL"
+            )},
+            actions = {'eks-kill-pod': aws_fis.CfnExperimentTemplate.ExperimentTemplateActionProperty(
+                action_id = "aws:eks:inject-kubernetes-custom-resource",
+                parameters = dict(
+                    kubernetesApiVersion = "chaos-mesh.org/v1alpha1",
+                    kubernetesKind = "PodChaos",
+                    kubernetesNamespace = "chaos-mesh",
+                    kubernetesSpec = json.dumps({
```

Comment:
This part can be changed with your manifests if you want to inject different fault scenarios into the kubernetes objects.